src/sdk/bucketspace_fs.js

```diff
@@ -155,6 +155,7 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
                     }
                 }
             }
+            bucket.stat = await this.stat_bucket(bucket.name.unwrap());
```

Comment:
the name of the function is confusing a bit because this function is in bucketspace_fs and not in config_fs
stat also can be on the storage path of the bucket
this reminds me there is a very confusion in the function call in read_bucket_sdk_info - 
`const is_valid = await this.check_bucket_config(bucket);`
check_bucket_config() stats the bucket_storage_path
if you can fix this issue as part of this PR it'll be great